int-tests/007-update.t

```diff
@@ -0,0 +1,29 @@
+This cram test performs the update command on one slice.
+
+Setup unit files
+
+  $ GROUP=007-update
+  $ mkdir $GROUP
+  $ echo "[Unit]\nDescription=Unit 1\n[Service]\nExecStart=/bin/bash -c 'while true; do echo Hello %n; sleep 10; done'\n" > $GROUP/$GROUP-1-unit@.service
+  $ echo "[Unit]\nDescription=Unit 2\n[Service]\nExecStart=/bin/bash -c 'while true; do echo Hello %n; sleep 10; done'\n[X-Fleet]\nMachineOf=$GROUP-1-unit@%i.service\n" > $GROUP/$GROUP-2-unit@.service
+
+Submit units
+
+  $ inagoctl --fleet-endpoint=${FLEET_ENDPOINT} submit $GROUP 1 >010.out 2>&1
+  $ inagoctl --fleet-endpoint=${FLEET_ENDPOINT} status $GROUP -v >021.out 2>&1
+  $ inagoctl --fleet-endpoint=${FLEET_ENDPOINT} status $GROUP >025.out 2>&1
+  $ inagoctl --fleet-endpoint=${FLEET_ENDPOINT} start $GROUP >030.out 2>&1
+  $ inagoctl --fleet-endpoint=${FLEET_ENDPOINT} status $GROUP -v >041.out 2>&1
+  $ inagoctl --fleet-endpoint=${FLEET_ENDPOINT} status $GROUP > 045.out  2>&1
+  $ sleep 10
+
+Modify unit and perform update
+  $ echo "[Unit]\nDescription=Unit 1 (CHANGED)\n[Service]\nExecStart=/bin/bash -c 'while true; do echo Hello %n; sleep 10; done'\n" > $GROUP/$GROUP-1-unit@.service
+  $ inagoctl --fleet-endpoint=${FLEET_ENDPOINT} update $GROUP --max-growth=1 --min-alive=1 > 48.out 2>&1
+  $ sleep 10
```

Comment:
@JosephSalisbury @zeisss maybe have more information? I would vote to remove it. Only makes the tests slow.